api/specs.proto

```diff
@@ -91,6 +91,7 @@ message ServiceSpec {
 // ReplicatedService sets the reconciliation target to certain number of replicas.
 message ReplicatedService {
 	uint64 replicas = 1;
+	repeated string task_remove = 2;
```

Comment:
Thank you for your feedback. I will update this pull request according to your feedback.